website/docs/reference/resource-configs/group.md

```diff
@@ -0,0 +1,300 @@
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+</VersionBlock>
+
+<File name='dbt_project.yml'>
+
+```yml
+models:
+  [<resource-path>](resource-path):
+    +group: finance
+```
+
+</File>
+
+<File name='seeds/properties.yml'>
+
+```yml
+seeds:
+  - name: [<seed-name>]
+    group: finance
+```
+
+</File>
+
+
+</TabItem>
+
+<TabItem value="snapshots">
+
+<VersionBlock lastVersion="1.4">
+
+Support for grouping snapshots was added in dbt Core v1.5
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+</VersionBlock>
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  [<resource-path>](resource-path):
+    +group: finance
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{% snapshot [snapshot_name](snapshot_name) %}
+
+{{ config(
+  group='finance'
+) }}
+
+select ...
+
+{% endsnapshot %}
+```
+
+</File>
+
+
+</TabItem>
+
+<TabItem value="tests">
+
+<VersionBlock lastVersion="1.4">
+
+Support for grouping tests was added in dbt Core v1.5
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+</VersionBlock>
+
+<File name='dbt_project.yml'>
+
+```yml
+tests:
+  [<resource-path>](resource-path):
+    +group: finance
+```
+
+</File>
+
+<File name='tests/properties.yml'>
+
+```yml
+version: 2
+
+<resource_type>:
+  - name: <resource_name>
+    tests:
+      - <test_name>:
+          config:
+            group: finance
+```
+
+</File>
+
+<File name='tests/<filename>.sql'>
+
+```sql
+{% test <testname>() %}
+
+{{ config(
+  group='finance'
+) }}
+
+select ...
+
+{% endtest %}
+```
+
+</File>
+
+<File name='tests/<filename>.sql'>
+
+
+```sql
+{{ config(
+  group='finance'
+) }}
+```
+
+</File>
+
+</TabItem>
+
+<TabItem value="analyses">
+
+<File name='analyses/<filename>.yml'>
+
+```yml
+version: 2
+
+analyses:
+  - name: <analysis_name>
+    group: finance
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="metrics">
+
+<VersionBlock lastVersion="1.4">
+
+Support for grouping metrics was added in dbt Core v1.5
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+</VersionBlock>
+
+<File name='dbt_project.yml'>
+
+```yaml
+metrics:
+  [<resource-path>](resource-path):
+    [+](plus-prefix)group: finance
+```
+
+</File>
+
+<File name='models/metrics.yml'>
+
+```yaml
+version: 2
+
+metrics:
+  - name: [<metric-name>]
+    group: finance
+
+```
+
+</File>
+
+</TabItem>
+
+</Tabs>
+
+## Definition
+An optional configuration for grouping models, analysis, snapshots, tests, and metrics. When a resource is grouped, dbt will allow it to reference private models within the same group.
+
+For more details on reference access between resources in groups, check out [model access](/docs/collaborate/publish/model-access#groups).
+
+## Examples
+### Prevent a 'marketing' group model from referencing a private 'finance' group model
+This could be useful if you want to prevent other groups from building on top on models that are rapidly changing, experimental, or otherwise internal to a group or team. 
```

Comment:
```suggestion
This is useful if you want to prevent other groups from building on top of models that are rapidly changing, experimental, or otherwise internal to a group or team. 
```